packages/react-components/src/components/BrowserPermissionDenied.tsx

```diff
@@ -94,16 +94,21 @@ const BrowserPermissionDeniedContainer = (props: BrowserPermissionDeniedProps):
         </Stack>
       </Stack>
       <Stack styles={textContainerStyles}>
-        <Text styles={primaryTextStyles}>{strings.primaryText}</Text>
-        <Text styles={secondaryTextStyles}>{strings.secondaryText}</Text>
-        <PrimaryButton
-          styles={mergeStyleSets(primaryButtonStyles, styles?.primaryButton)}
-          text={strings.primaryButtonText}
-          onClick={onTryAgainClick}
-        />
-        <Link styles={mergeStyleSets(linkTextStyles, styles?.troubleshootingLink)} onClick={onTroubleshootingClick}>
-          {strings.linkText}
-        </Link>
+        <Text styles={primaryTextStyles}>{strings?.primaryText}</Text>
+        <Text styles={secondaryTextStyles}>{strings?.secondaryText}</Text>
+        {onTryAgainClick && (
+          <PrimaryButton
```

Comment:
Same as https://github.com/Azure/communication-ui-library/pull/2434/files#diff-86f3474befa4ad506ce8c7c7f9d1de9345eb84c5a14726dc9935af0180f0ccdcR91 , each of these needs to check if the string is valid. If not the react element should not be rendered